src/ui/Tooltip.tsx

```diff
@@ -0,0 +1,27 @@
+import { OverlayTrigger } from 'react-bootstrap';
+import { Tooltip as BootstrapTooltip } from 'react-bootstrap';
+
+import { QuestionCircleIcon } from '@/pages/NewFeatureTooltip';
+
+type TooltipPlacement = 'top' | 'right' | 'bottom' | 'left';
+
+type Props = {
+  id: string;
+  tooltip: string;
```

Comment:
```suggestion
  content: string;
```
Maybe content is a bit more clear